test/helpers/WorldHelper.cpp

```diff
@@ -180,24 +171,28 @@ proto::World WorldHelper::getWorldMsg(int amountUs, int amountThem, bool withBal
         msg.mutable_yellow()->CopyFrom(randomYellow);
         msg.mutable_blue()->CopyFrom(randomBlue);
 
-        if (withBall) msg.set_allocated_ball(&randomBall);
-    } while (!allPositionsAreValid(msg, true));
+        if (withBall) {
+            std::cerr << "[WorldHelper] Caution: generating a world with a ball is not stable!" << std::endl;
```

Comment:
I am confused by this line. What do you mean, not stable?

Reply:
I  mean that it sometimes gives a segfault. (I couldn't find the problem quickly and it's outside the scope of this PR). It needs an issue.